---
layout: archive
title: "Research"
permalink: /research/
author_profile: true

---

{% include base_path %}

Our research focuses on two overall questions:

Q1: How do normal, healthy cells evolve to become cancer cells?

Q2: How do we translate predicted cancer risk to patients effectively?

These two questions have traditionally been examined in isolation from one another. However, molecular and population data available are inextricably linked - the underlying dynamic process of cancer evolution manifests in the human body and measurements may be naturally reflected in incidence or in collected patient snapshots based on the timing of clinical recommmendations/study design, and interventions in practice.

We leverage these inherent connections by using mechanistic models, machine learning, and bioinformatics methods to integrate epidemiological, clinico-pathological and molecular cancer data in order to test and validate hypotheses. With a current focus on gastrointestinal cancers, we seek a coherent understanding of the carcinogenesis process so we may find and identify dangerous precursors (or diverse hotbeds of clones) in patients and forecast outcomes reliably.


![logo-right](/images/QCC_motivation_simple_14July2020.png){: .align-right}

The following are some examples of our collaborative research in GI cancer control:

## *What* are the premalignant alterations that are predictive of future cancers? 
<h3>Methods: </h3>
  Genome sequencing, bioinformatics, phylogenetics, biostatistics, predictive modeling, longitudinal data analysis
<h3>Reference links with applications: </h3>
1. [Copy number alterations precede cancers in inflammatory bowel disease](https://gut.bmj.com/content/68/6/985.abstract)
2. [Efficacy of screening modalities for detecting high-risk dysplasia in Barrett's esophagus](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004272)

## *When* are patients most at risk of premalignancy and would benefit from screening?
<h3>Methods: </h3>
  Mechanistic modeling, stochastic modeling, Bayesian inference of biological aging, cost-effectiveness analyses
<h3>Reference links with applications: </h3>
1. [Patient-specific molecular clocks for Barrett's esophagus onset ages](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004919)
2. [Optimal timing of screening and adaptive surveillance](https://www.biorxiv.org/content/10.1101/2020.02.11.927475v1)
3. [Estimating timescales of (pre)malignant growth from incidence curves](http://cancerres.aacrjournals.org/cgi/pmidlookup?view=long&amp;pmid=23054397)


## *Who* should we refer for screening and who then requires further interventions? 
<h3>Methods: </h3>
  Risk stratification, translational bioinformatics, evolutionary biomarkers, clinical decision support tools
<h3>Reference links with applications: </h3>
1. [UC-CaRE: a validated colitis-associated colorectal cancer risk prediction web-tool](https://www.medrxiv.org/content/10.1101/2020.04.10.20057869v1)
2. [Modeling suggests Barrett's esopohagus patients constitute most future esophageal adenocarcinomas](https://www.biorxiv.org/content/10.1101/2020.05.14.096826v1.abstract)


For further reading about our research, please see our section on "Cancer screening and early detection with modeling" as part of the [2019 Mathematical Oncology Roadmap](https://iopscience.iop.org/article/10.1088/1478-3975/ab1a09/meta).



